### ISSUE TITLE
Pimp your codebase & scale it!

### ISSUE THREAD

**Issue by cyp3rius** [WrocTypeScript/talks#68]
- **Your full name**: Mateusz Ziarko
- **Target audience** (beginner/intermediate/advanced/everyone): intermediate/advanced
- **Estimated duration**: 30 - 45 minutes
- **Keywords**: MicroFrontends, Microservices, Serverless, Full Stack, Scalability

# Pimp your codebase & scale it!

### One-sentence summary
How we may approach the topic of refreshing / rewriting / splitting our monolithic application when it is impossible to start everything again?

### What's the format — is it a case study, a live coding session, a workshop or something else?
It's a concepts discussion speech, a bit of examples and possible approached to the topic

### Tell us more about the talk
How we may approach the topic of refreshing / rewriting / splitting our monolithic application when it is impossible to start everything again? A few words about Lambdas / Functions, Micro Services in our ecosystem, as well as about the approach to solving this problem on the client side using the concept of Micro Front'ends

### A few words about yourself

Engineering Manager @ VirtusLab who is building and leading teams for projects of various scale. A Full Stack Developer from origin and passion, for whom the concept of 'business day' does not apply, because passion is can't be limited.

### How can we find you on social media?
FB: https://www.facebook.com/mateusz.ziarko
LN: https://www.linkedin.com/in/mateuszziarko/

### Would you be willing to have a Q/A session after the talk?
Yes

### Do you mind if we record the event?
I'm ok with it

### Is there anything we can help you with?
In case of need I'll shout :)

**Comment by hasparus** [comment]
Hey Mateusz! Thanks for the issue. We've spoken offline to add it to April's agenda.

It will surely be fascinating, considering the fact that Majestic Monolith was mentioned on the meetup this Wednesday!

![image](https://user-images.githubusercontent.com/15332326/75526947-06039d00-5a13-11ea-8f7e-5b8c3450366a.png)

**Comment by cyp3rius** [comment]
Sounds like perfect followup :)

**Comment by hasparus** [comment]
Hey Mateusz!

We’ve recently became an online meetup, due to well known circumstances. Are you okay with giving your talk online on our YouTube stream?

**Comment by cyp3rius** [comment]
@hasparus sure, I've got no problem with YT stream, all live I assume? :)

**Comment by hasparus** [comment]
yeah, we did the [last one](https://www.youtube.com/watch?v=2fHBBiBGYio) this way. It went nicely.
The chat is a nice feature, though the speaker doesn't see it, because it's super distracting.
We'll all you and @mstrag on hangouts or zoom and the mc will stream to the YouTube channel.
